apps/frontend/src/libs/modules/toast/toast.ts

```diff
@@ -0,0 +1,2 @@
+export { Toast } from "./toast.module.js";
+export { toast } from "react-toastify";
```

Comment:
Create a new module in ~/libs/modules and add there toast notifications. We should have separate methods for error, warning and others, which will receive a message